metron-interface/metron-alerts/src/app/model/search-request.ts

```diff
@@ -0,0 +1,7 @@
+export class SearchRequest {
+  _source: string[];
+  query = { query_string: { query: '' } };
```

Comment:
Shouldn't this type by 'string'?  This doesn't match the model in the PR comments.